Header header
int16 reward
int16 iteration_num
bool reset_world
int16 robot_location
